docs/getting-started/integrations/angular.md

```diff
@@ -703,6 +703,45 @@ The `moduleResolution` option of the TypeScript configuration determines the alg
 * You can update Angular to version 18, where the `moduleResolution` option is set to `bundler`  by default.
```

Comment:
Earlier, there was only one known issue, so it didn't make sense to add a subheading. But now, we have two known issues, so I would add a meaningful heading for the first known issue above, too. Something like: "Module resolution issues".